lib/dialects/mssql/schema/mssql-columncompiler.js

```diff
@@ -63,6 +65,36 @@ class ColumnCompiler_MSSQL extends ColumnCompiler {
     return '';
   }
 
+  defaultTo(value, constraintName) {
+    if (value === void 0) {
+      return '';
+    } else if (value === null) {
+      value = 'null';
+    } else if (value instanceof Raw) {
```

Comment:
There is no need to import raw here, 
```
if (value && value.isRawInstance) {
```
should suffice

Reply:
Or use `formatDefault()` from `lib/formatter/formatterUtils.js`

Reply:
You'd need to pull latest changes from master for that, though

Reply:
I've just lifted this from the base definition. Happy to change this but should it be changed in the base too?

Reply:
Pull from the master, base was adjusted to use `formatDefault` :)

Reply:
Ah, not sure why I wasn't up to date, I've updated the PR to use `formatDefault` now